status/raw/status-app-protocols.md

```diff
@@ -0,0 +1,373 @@
+---
+title: STATUS-PROTOCOLS
+name: Status Protocol Stack
+status: raw
+category: Standards Track
+description: Specifies the Status application protocol stack
+editor: Hanno Cornelius <hanno@status.im>
+contributors: 
+- Jimmy Debe <jimmy@status.im>
+- Aaryamann Challani <p1ge0nh8er@proton.me>
+
+---
+
+## Abstract
+
+This specification describes the Status Application protocol stack.
+It focuses on elements and features in the protocol stack for all application-level functions:
+- functional scope (also _broadcast audience_)
+- content topic
+- ephemerality
+- end-to-end reliability layer
+- encryption layer
+- transport layer (Waku)
+
+It also introduces strategies to restrict resource usage, distribute large messages, etc.
+Application-level functions are out of scope and specified separately. See:
+- [55/STATUS-1TO1-CHAT](../55/1to1-chat.md)
+- [56/STATUS-COMMUNITIES](../56/communities.md)
+
+## Status protocol stack
+
```

Comment:
```suggestion
The keywords “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”,
“SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and
“OPTIONAL” in this document are to be interpreted as described in [2119](https://www.ietf.org/rfc/rfc2119.txt).
```
Disclaimer could be placed here since this is used within the document.